app/views/availability_zone/show.html.haml

```diff
@@ -3,7 +3,7 @@
     = render :partial => "layouts/performance_async"
   - else
     - if %w(instances cloud_volumes).include?(@display) && @showtype != "compare"
-      = render :partial => "layouts/gtl", :locals => {:action_url => "show/#{@availability_zone.id}"}
+      = render :partial => "layouts/gtl", :locals => {:action_url => "show/#{@record.id}"}
```

Comment:
as we talked we miss the helper method for that issue right now, this building of the ```:action_url``` is used across the lot of views